### PR TITLE
allow zooming and page navigation in pdf view using short cuts

### PR DESCRIPTION
**Pull request by nittka**
These changes allow zooming and page navigation using keyboard short cuts in the pdf view. Ideally, I'd like to put all zooming actions into a "Zoom" drop down menu in order to gain space for issue #4 (this should have no negative impact on usability if short cuts are available). There might be better default key bindings for the "fit to" actions...

### CODE REVIEW

**Comment by nittka** [comment]
The first commit does the key binding only, the second one puts all zoom actions into a dropdown.

File: org.eclipse.ui.views.pdf/plugin.xml [hunk 1]
```diff
@@ -49,4 +49,110 @@
...
+           id="org.eclipse.ui.views.pdf.NavigateNext"
+           name="Next Page">
+     </command>
+  </extension>
+  <extension
+        point="org.eclipse.ui.bindings">
+     <key
+           commandId="org.eclipse.ui.views.pdf.ZoomIn"
+           schemeId="org.eclipse.ui.defaultAcceleratorConfiguration"
+           sequence="CTRL++">
```
Comment (thSoft) [inline org.eclipse.ui.views.pdf/plugin.xml]: Please substitute CTRL with M1 for platform independence (it corresponds to CMD on OS X).

**Comment by nittka** [comment]
The binding has been modified according to the comment. Further view related tickets have been addressed in separate commits.

**Comment by thSoft** [comment]
I recommend [this](https://github.com/yusukekamiyamane/fugue-icons/blob/master/icons-shadowless/documents-stack.png) or [this](https://github.com/yusukekamiyamane/fugue-icons/blob/master/icons-shadowless/documents-text.png) for the file list icon, and also propose renaming it to documents.png or similar. Otherwise, these are useful improvements!

**Comment by nittka** [comment]
Will you add the appropriate license information?

**Comment by thSoft** [comment]
Done: https://github.com/thSoft/eclipse-commons/commit/37629285e2096714ee7ae816474a601e9d9a9972

**Comment by nittka** [comment]
The icon has already been replaced, so everything is set for merging?!

**Pull request merged**